bundles/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/tool/CreateTTSCache.java

```diff
@@ -119,17 +118,29 @@ private void generateCacheForMessage(String apiKey, String cacheDir, String loca
             String format, String msg) throws IOException {
         String trimmedMsg = msg.trim();
         if (trimmedMsg.length() == 0) {
-            System.err.println("Ignore msg=''");
+            systemErrLog("Ignore msg=''");
             return;
         }
         try {
             CachedVoiceRSSCloudImpl impl = new CachedVoiceRSSCloudImpl(cacheDir, false);
             File cachedFile = impl.getTextToSpeechAsFile(apiKey, trimmedMsg, locale, voice, codec, format);
-            System.out.println("Created cached audio for locale='" + locale + "', voice='" + voice + "', msg='"
-                    + trimmedMsg + "' to file=" + cachedFile);
+            systemOutLog("Created cached audio for locale='" + locale + "', voice='" + voice + "', msg='" + trimmedMsg
+                    + "' to file=" + cachedFile);
         } catch (IllegalStateException | IOException ex) {
-            System.err.println("Failed to create cached audio for locale='" + locale + "', voice='" + voice + "',msg='"
+            systemErrLog("Failed to create cached audio for locale='" + locale + "', voice='" + voice + "',msg='"
                     + trimmedMsg + "'");
         }
     }
+
+    private void systemOutLog() {
+        System.out.println();
+    }
+
+    private void systemOutLog(String text) {
+        System.out.println(text);
+    }
+
+    private void systemErrLog(String text) {
+        System.err.println(text);
+    }
```

Comment:
Are these wrappers really needed, they don't seem to add anything, except adding a layer to something quite standard and known? Second question, any reason to call `System.out.println()` for each line - perhaps it would be simpler (and more performant) to call `System.out.print()` with a formatted string containing all the lines to be output?

Reply:
The only reason was to reduce the number of warnings when compiling. Should I revert ?

Yes, it should be possible, I think, to have only one unique big stirrng with `\n`.

Reply:
Ahh, that's why... nullness annotations. So the fix would be something like this:

```
    private void usage() {
        PrintStream printStream = System.out;
        if (printStream == null) {
            return;
        }
        printStream.println("Usage: java org.openhab.voice.voicerss.tool.CreateTTSCache <args>");
```

Probably not worth to output as a single string.

Reply:
Fixed.
I don't know why I did not think about that !